src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+    trademark_list: 商標リスト
+    cookies: クッキーポリシー
+
+#------------------------------------------------------------------------------
+# 404 Page
+#------------------------------------------------------------------------------
+
+404_page:
+  title: 404エラー
+  subtitle: ページが見つかりません
+  description: 申し訳ございませんが、お探しのページは、移動または消除された可能性があります。
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメンテーションを読む
+
+#------------------------------------------------------------------------------
+# Branding Page
+#------------------------------------------------------------------------------
+
+branding_page:
+  title: ブランディング
+  description: 正しいロゴの使い方
+  sections:
+    name:
+      title: 名前
+      description: >
+        ESLint を書くときは、E,SとLは大文字を使用して下さい。「ES」はECMAScriptの略称、Lは「int」の頭文字です。
+    logo:
+      title: ロゴ
+      description:
+        - >
+          ESLintのロゴは、様々な背景に配置することができます。十分な縦・横のパディングが必要です。
+        - >
+          隙間からなる内側の六角形の大きさの2倍が、ロゴとワードマークの間隔、およびロゴの周囲の余白の
+          大きさの目安になります。
+        - >
+          我々のロゴは汎用性があります。様々なブランドカラーに使用することができ、白以外の背景では透過させることで
+          明るい色を模倣することができます。ESLintのロゴは、ワードマークを付けずに単独で使用することもできますが、
+          可能であればESLintのロゴとワードマークの両方を使用することが望ましいです。
+      download_svg: SVG ダウンロード
+      download_png: PNG ダウンロード
+      change_background: "背景色の変更"
+    color_palette:
+      title: カラーパレット
+      description:
+        - >
+          ロゴや文言と同じくらい、ESLintのカラーパレットにはブランドを表す力があります。
+          デザインの見た目に影響を与えるだけでなく、感情を呼び起こし、
+          ESLintブランドの個性を反映することができるのです。
+        - >
+          第一のブランドカラーはボタン、リンク、入力コンポネントなどすべてのインタラクティブな要素で使用されます。
+          ロゴに由来しています。既存の2色を使って、独自の一次色調を作り出しています。
+      brand_palette: ブランド　パレット
+      colors:
+        primary: プライマリ
+        white: 白色
+        neutral: ニュートラル
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメンテーションを読む
+
+#------------------------------------------------------------------------------
+# Homepage
+#------------------------------------------------------------------------------
+
+homepage:
+  title: JavaScriptコードの問題点の発見と修正
+  description: >
+    ESLintは、コードを静的に解析し、問題を迅速に発見します。
+    ほとんどのテキストエディタに組み込まれており、CIパイプラインの一部としてESLintを実行することができます。
+
+  animation:
+    enabled: true
+    text: Find and fix problems in your JavaScript code
+    highlight_word: problems
+    wrong_word: problmes
+
+  install:
+    title: ESLint インストールコマンド
+    copy: インストールコマンドをコピーします
+    copied: コピーしました
+
+  versions:
+    title: ESLintバージョン
+    latest: 最新バージョン
+    upcoming: 次のバージョン
+    development: 開発中バージョン
+    dateline: <span class="text-dark">VERSION</span>（DATE）
+
+  explainer:
+    image:
+      src: eslint-features-img.png
+      alt: "ESLintが動作しているVisual Studioのコードウィンドウのスクリーンショット：IDE内のESLintエラーに下線を引いています。"
+    title: The pluggable linting utility for JavaScript and JSX
+    description: >
+      ESLintは、あなたのJavaScriptコードの問題を発見し、修正することを支援するオープンソースプロジェクトです。
+      ブラウザでJavaScriptを書いているか、サーバで書いているか、フレームワークを使っているか、使っていないかに関係なく、
+      ESLintは以下のことを支援します。あなたのコードが最高の状態で生きられるように。
```

Original
> ESLint is an open source project that helps you find and fix problems with your JavaScript code. It doesn't matter if you're writing JavaScript in the browser or on the server, with or without a framework, ESLint can help your code live its best life.

https://github.com/eslint/new.eslint.org/blob/main/src/_data/sites/en.yml#L208-L212

_live its best life_ is a kind of idiom. Idioms are difficult to translate as they are. Therefore, at first I replace it with a plain English expression.

_ESLint help to keep your code good state._

My suggestion is:
```suggestion
      ブラウザのためか、サーバのためか、フレームワークを使用しているかに関わらず、ESLintはあなたのコードが
      良い状態になるように支援します。
```